### DISCUSSION TITLE
Next js app to one bundle.js

### DISCUSSION THREAD

**Discussion by lkowalewski** [vercel/next.js#12423] [Help]
Hello 
How to build applications for one bundle.js file ?

**Comment by lfvjimisola** [comment]
Our team is wondering about this as well.

We are currently doing `next build && next export` to export static HTML but would like for it to create **a single** file (e.g. bundle.js) as described by Webpack [here](https://webpack.js.org/concepts/output/#usage).

There are some tickets on this topic but they don't seem to apply 100%: https://github.com/vercel/next.js/issues/5708
